v2/manager.go

```diff
@@ -572,6 +589,13 @@ func (c *Manager) MemoryEventFD() (int, uint32, error) {
 		syscall.Close(fd)
 		return 0, 0, fmt.Errorf("failed to add inotify watch for %q", fpath)
 	}
+	// monitor to detect process exit/cgroup deletion
+	evpath := filepath.Join(c.path, "cgroup.events")
+	wd_cg, err := syscall.InotifyAddWatch(fd, evpath, unix.IN_MODIFY)
```

Comment:
- err seems unchecked
- nit: underscores are usually avoided for variable names in Go

Reply:
Thanks, didn't know the thing about underscores. Added error checking in both `InotifyAddWatch` calls, and replaced `wd_cg` with `_` since it is discarded anyway.